docker/docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '3.5'
```

Comment:
maybe just a comment here indicating what this file actually does?

Reply:
maybe more generally indicating how these four files work together - in the README file?

Reply:
Yes - I had realised it needed something more in the documentation.  Of course I have now forgotten what I did and why...I'll do that over the next few days.